### DISCUSSION TITLE
Session vs make:auth

### DISCUSSION THREAD

**Discussion by yahyaoui-mohamed** [symfony/symfony#45341] [Q&A] [answered]
**Hello, i'm currently working on a project and i need to make a login form, so i'm asking if is it better to work with sessions or the built-in make:auth in symfony 6?**

**Comment by ahundiak** [comment, marked as answer]
Is your project based in the Symfony 6 framework?  If so then yes you will probably want to use the built in authentication/authorization capabilities.  Probably want to run make:user, make:auth and make:registration-form to hopefully get a working example.

If you are not actually using the framework then I would suggest staying far far away from the Symfony security components.

**Reply by yahyaoui-mohamed** [reply]
Thank you so much.
